jbpm-test-util/src/main/java/org/jbpm/test/persistence/scripts/quartzdialects/QuartzDialectResolver.java

```diff
@@ -30,6 +30,7 @@
     private static Map<String, Dialect> DIALECT_BY_NAME = new HashMap<>();
 
     {
+        registerDialect(new MySQLCustomDialect(), "MariaDB");
```

Comment:
Wouldn't be clearer to use (or create) a new custom dialect for MariaDB instead of using `MySQLCustomDialect` to decoupling?

Reply:
No, it's not worth it. MariaDB is 100% compatible with mysql and currently we have only scripts for mysql